test/mocha/glob.js

```diff
@@ -21,4 +21,14 @@ describe("minify() with input file globs", function() {
         });
         assert.strictEqual(result.code, 'var print=console.log.bind(console);print("qux",function(n){return 3*n}(3),function(n){return n/2}(12)),function(n){print("Foo:",2*n)}(11);');
     });
+    it("should throw with non-matching glob string", function() {
+        assert.throws(function() {
+            Uglify.minify("test/input/issue-1242/blah.*");
+        }, "should throw file not found");
```

Comment:
> By "safeguard" do you mean adding a test?

Yes 😅

May be I should check the error message to make this whole thing more clear?

I can also add a test that explicitly call `simple_glob()` to make sure it does what you expected from UNIX.